lib/loaders/loaders_with_authentication/index.js

```diff
@@ -32,6 +32,7 @@ export default (accessToken, userID, requestID) => {
       "is_saved"
     ),
     saleArtworksLoader: gravityLoader(id => `sale/${id}/sale_artworks`),
+    suggestedArtistsLoader: gravityLoader("me/suggested/artists", {}, { headers: true }),
```

Comment:
We want to be able to pluck the total count, so make sure we are able to resolve headers.